project/SparkBuild.scala

```diff
@@ -109,6 +109,16 @@ object SparkBuild extends PomBuild {
     if (profiles.contains("jdwp-test-debug")) {
       sys.props.put("test.jdwp.enabled", "true")
     }
+    if (profiles.contains("user-defined-pb")) {
```

Comment:
also cc @JiangHaonan

Reply:
Is this from the pom file?

Reply:
It is customary that the profile name should be consistent with that in `pom.xml`.

The `profiles` from env `SBT_MAVEN_PROFILES`,

https://github.com/apache/spark/blob/84fabc27d688601feabb42abfc7356cd743b3c38/build/sbt-launch-lib.bash#L98-L102